public/components/event_analytics/home/home.tsx

```diff
@@ -279,7 +280,6 @@ export const Home = (props: IHomeProps) => {
       setToast(`Sample events added successfully.`);
     } catch (error: any) {
       setToast(`Cannot add sample events data, error: ${error}`, 'danger');
-      console.error(error.body.message);
```

Comment:
Any reason to remove the `console.error`?

Reply:
Linting doesn't allow console.xxx